prep-javascript-variables/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullName = 'Alexandra Christensen';
+    console.log('Value of full Name:', fullName);
+    var numberOfPets = '1';
+    console.log('Value of numberofPets:', numberOfPets);
```

Comment:
We'll want to leave the quotes off of our number values so that the javascript engine doesn't interpret them as strings

Reply:
hi! i made the fix here. please let me know how it looks. thanks!